Meta/CLion/run.sh

```diff
@@ -1,22 +1,23 @@
 #!/bin/sh
-# shellcheck disable=SC2086 # FIXME: fix these globing warnings
 
 set -e
 
-#SERENITY_PACKET_LOGGING_ARG="-object filter-dump,id=hue,netdev=breh,file=e1000.pcap"
+: "${SERENITY_RAM_SIZE:='256M'}"
+export SERENITY_RAM_SIZE
 
-[ -e /dev/kvm ] && [ -r /dev/kvm ] && [ -w /dev/kvm ] && SERENITY_KVM_ARG="-enable-kvm"
+# set SERENITY_ROOT to the path to the serenity directory
+: "${SERENITY_ROOT:='/mnt/c/Users/Ragnarok/serenity-project/serenity'}"
```

Comment:
😐 I don't think this is the way to allow running run.sh from anywhere

By the way, you can already do exactly that, you only need to set `SERENITY_ROOT` properly, so...

Reply:
Or — perhaps I'm mistaken and you cannot run `run.sh` from anywhere, if it expects to find `_disk_image` and `Kernel/Kernel` in the current directory.

The question then is why would you want that?

Reply:
That's the CLion version of `run.sh`, it used to be a copy of the standard `run.sh` with the default values of the variables changed.
That didn't make much sense: you can just override the value of the environment variables and then run the standard `run.sh` from that file. 

`run.sh` expected to find `./Kernel/Kernel` and `./_disk_image`; when running the script with the `b` (`bochs`) argument, it expects to find a bochs configuration file in `./.bochsrc`.

When using `run.sh` with the `qgrub` or the `b` option, we should use `grub_disk_image` as the disk image instead of using `_disk_image`.

> The question then is why would you want that?

Why not? The `bochs` configuration file is not in the build directory and we are not copying it there; since we have to use an absolute path to that file, let's also allow running `run.sh` anywhere.

If you want to run `run.sh` on the current directory, you can still do it: `SERENITY_BUILD= /path/to/run.sh`.

Reply:
You still have to pass `run.sh` the directory where to look for `_disk_image` and the kernel for, *somehow*. I feel using the current directory for this is a nice, Unix-y way — better than using an environment variable. But perhaps you can convince me otherwise. Are there any cases where it's convenient to pass an environment variable, but inconvenient to set the current directory, for example?

Reply:
Oh, and hardcoding a path like this is certainly a wrong thing to do 😀

Reply:
> Oh, and hardcoding a path like this is certainly a wrong thing to do

What path are you refering to? `/mnt/c/Users/Ragnarok/serenity-project/serenity` ? That was already there (in `Meta/CLion/run.sh`).

I totally agree with you!

I just rewrote that script to use `Meta/run.sh`.

Reply:
> You still have to pass run.sh the directory where to look for _disk_image and the kernel for, somehow.

Not really unless we use `-` options. `run.sh` already has arguments: `/path/run.sh b` emulates serenity using `bochs` for example, or `/path/to/run.sh qcmd boot_mode=text another_argument hello`, emulates serenity in qemu passing to the kernel `boot_mode=text`, `another_argument` and `hello` as command line argument.

I can just default `run.sh` to look in `.` for `_disk_image`, `grub_disk_image` and `Kernel/Kernel`, but to look on "$SERENITY_ROOT/Meta` for the `bochs` configuration file (`bochsrc`), unless SERENITY_BUILD is specified if you want.

Reply:
ping @bugaevc

Reply:
> I can just default `run.sh` to look in `.` for `_disk_image`, `grub_disk_image` and `Kernel/Kernel` unless `SERENITY_BUILD` is set as an environment variable, but to look in `"$SERENITY_ROOT/Meta"` for the `bochs` configuration file (`bochsrc`) if you want.

Yep, that sounds good to me